lib/logasm/adapters/stdout_adapter.coffee

```diff
@@ -1,23 +1,24 @@
 winston = require('winston')
-_  = require 'underscore'
+buildEvent = require('../event')
 
 class StdoutAdapter
-  constructor:(level) ->
+  constructor:(level, _serviceName, options) ->
     @logger = new winston.Logger
     @logger.add winston.transports.Console,
-      timestamp: ->
-        "#{new Date().toISOString()} ##{process.pid}"
+      json: Boolean(options.json)
+      timestamp: !Boolean(options.json) # Json adds @timestamp manually
```

Comment:
the default timestamp is okay. Process pid is not useful at all with kubernetes (all processes are pid 0). I would say it wasn't even useful with the previous deployment.